src/utils/api.jsx

```diff
@@ -27,3 +27,16 @@ export const getCommentsByArticleID = (article_id) => {
     return response.data.comments;
   });
 };
+
+export const incrementArticleVotes = (article_id, newVote) => {
+  return api.patch(`/api/articles/${article_id}`, { inc_votes: newVote}).then((response) => {
+      return response.data;
+    
+  });
+};
+
+export const addCommentToArticleByID = (article_id, newComment) => {
+  return api.post(`/api/articles/${article_id}/comments`, newComment).then ((response) => {
+    return response.data.comment;
+  })
```

Comment:
I'm a fan of using this separate api file and extracting the data you need here instead of in the component 👍